keyboards/dztech/dz65rgb/keymaps/chocol8/keymap.c

```diff
@@ -0,0 +1,55 @@
+#include QMK_KEYBOARD_H
+
+enum tofu68_layers {
+	_MAQWERTY,
```

Comment:
Indenting is off here.  

Not sure if this is intentional or due to a mix of spaces and tabs.

Reply:
fixed in new commit(s) @drashna